tests/test_port_config.py

```diff
@@ -197,7 +197,7 @@ def test_recirc_port(self, dvs):
         dvs.start_swss()
         time.sleep(5)
 
-        polling_config = PollingConfig(polling_interval=0.1, timeout=15, strict=True)
+        polling_config = PollingConfig(polling_interval=0.1, timeout=20.00, strict=True)
```

Comment:
Why increase the timeout here? Can you elaborate why this dhcp rate limiting change needs increased timeout for these test cases?

Reply:
reverted timeout. The timeout was added because, during code failures, the logs indicated that the DVS was not ready and needed more time. This adjustment was made to address the issue